docs/configuration.asciidoc

```diff
@@ -148,6 +148,27 @@ NOTE: The service name must conform to this regular expression: `^[a-zA-Z0-9 _-]
 | Name of the entry assembly       | String 
 |============
 
+[float]
+[[config-service-node-name]]
+==== `ServiceNodeName` (added[1.2])
```

Comment:
I personally linked the [Go version of this doc](https://github.com/elastic/apm-agent-go/pull/651/files#diff-01c1a79a5a1e1b67003a1885365f2cf5R153) better, but this is also totally fine.